### PR TITLE
Init. of missing docs up to the letter d

### PR DESCRIPTION
**Pull request by pp-r**
Create docs
app_dispatch
app_domoticz
app_droppy

### CODE REVIEW

File: pages/04.applications/10.docs/domoticz/app_domoticz.fr.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    créez un domaine ou sous-domaine pour recevoir les informations (par exemple : 'mqtt.your.domain.tld')
+    connecter vous en ligne de commande à votre serveur
+    taper la commande suivante : yunohost app setting domoticz mqtt_domain -v mqtt.your.domain.tld
+    Procédez à la mise à jour. Si vous êtes déjà sur la dernière version, utiliser la commmande suivante : yunohost app upgrade domoticz --force
+
+### Configuration
+
+Sensors, language and this kind of stuff
+
+Main configuration of the app take place inside the app itself.
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/domoticz/app_domoticz.fr.md:85]:
```suggestion
```

File: pages/04.applications/10.docs/domoticz/app_domoticz.fr.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+#### Senseurs, langue et ce genre de choses
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/domoticz/app_domoticz.fr.md:87]:
```suggestion
#### Capteurs, langue...
```

File: pages/04.applications/10.docs/domoticz/app_domoticz.fr.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Toute la configuration de l'application a lieu dans l'application elle même
+
+#### Access et API
+
+Par défaut, l'accès aux API JSON est autorisé sur cette URL /votredomaine.tld/api_/chemindedomoticz. Donc, si vous accédez à domoticz par https://votredomaine.tld/domoticz, utilisez le chemin suivant pour l'api: /votredomaine.tld/api_/domoticz/json.htm?votrecommandeapi
+
+Par défaut, seuls la mise à jour de senseur et les interrupteurs sont autorisés. Pour autoriser une nouvelle commande, vous devez (pour l'instant) manuellement éditer le fichier de configuration nginx :
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/domoticz/app_domoticz.fr.md:95]:
```suggestion
Par défaut, seuls la mise à jour de capteur et les interrupteurs sont autorisés. Pour autoriser une nouvelle commande, vous devez (pour l'instant) manuellement éditer le fichier de configuration NGINX :
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
Not reviewed until the end but you get the spirit:
- Caps when needed
- space before `:` in French
- ` ``` ` used around lines of code or path

File: pages/04.applications/10.docs/domoticz/app_domoticz.fr.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+#### Broker MQTT Mosquitto
+
+A l'installation, un broker MQTT, Mosquitto, est installé en même temps que Domoticz. La version installée est celle du dépot officiel du projet, et non des dépots Debian. Ce broker nécessite un domaine ou un sous-domaine particulier pour fonctionner (ex : mqtt.your.domain.tld) : il est nécessaire de créer ce domaine auparavant.
+
+##### Ajout dans domoticz
+
+Pour pouvoir l'utiliser, vous devez paramétrer la communication avec entre domoticz et le broker en suivant la documentation de domoticz dans la partie Add hardware "MQTT Client Gateway" Les users et mot de passe du broker sont automatiquement générés lors de l'installation. Vous pouvez les récupérer avec
+
+sudo yunohost app setting domoticz mqtt_user
+sudo yunohost app setting domoticz mqtt_pwd
```
[the remark this answers is not in the document]
Comment (pp-r) [reply pages/04.applications/10.docs/domoticz/app_domoticz.fr.md:47]: Ok, i'll try to not forget

**Comment by pp-r** [comment]
For Domoticz, which was much longer than the others, I did a lot of copy-pasting, I'll try to be less mechanical and automatic for the next ones.

**Comment by pp-r** [comment]
Thank you very much for the reviews !

File: pages/04.applications/10.docs/droppy/app_droppy.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+*Droppy* is a self-hosted file storage server with a web interface and capabilities to edit files and view media directly in the browser.
+
+### Disclaimers / important information
+
+Development of Droppy has been abandoned by its author, and its repository has been archived. Expect no future updates nor security fixes.
+
+### Configuration
+
+How to configure this app: a JSON configuration file with SSH. /var/www/droppy/config/config.json.
+Shared files are stored in the following directory: /home/yunohost.app/droppy/files
```
Comment (ericgaspar) [inline pages/04.applications/10.docs/droppy/app_droppy.md:25]:
```suggestion
Shared files are stored in the following directory: `/home/yunohost.app/droppy/files`
```

**Review by pp-r — COMMENTED** [review]
Thanks again for your patience as I don't have a solid knowledge of this tool and how to correct some things afterwards

**Review by pp-r — COMMENTED** [review]
Thanks !

**Pull request merged**